src/app/page.tsx

```diff
@@ -29,7 +28,7 @@ export default function Home() {
         </li>
         <li>
           <a
-            href="https://github.com/Mintbase/ref-finance-agent-next"
+            href="https://github.com/Mintbase/ref-finance-agent"
```

Comment:
lgtm, you will need to change the repo name as well, right?

Reply:
Yeah, pending approval and merge we should also change repo & vercel project name

Reply:
@sainthiago  Reverted that change until repo is updated